stl/inc/vector

```diff
@@ -4004,6 +4004,48 @@ _CONSTEXPR20 _OutIt _Copy_vbool(_VbIt _First, _VbIt _Last, _OutIt _Dest) {
     return _DestEnd;
 }
 
+template <class _VbIt, class _OutIt, class _Mapped_fn>
+_CONSTEXPR20 _OutIt _Transform_vbool_aligned(

```

Comment:
I moved this out to `<vector>` from `<algorithms>` because other algorithms are moved out.
However, I'm not sure if it is useful. 

For accessing `vector<bool>` representation it is not strictly necessary. Most of things are template-dependent member functions and datas. The only exception is `_Vbase`, which can be still deduced from iterators.

For throughput it does not look useful either. `<vector>` is more frequent than `<algorithm>` so it appears more useful to off-load `<vector>` instead.

For reference, 0f24d45309f2282fe296a95ee843bb084b6130d6 is the commit where this movement was made.